filament/backend/src/opengl/PlatformCocoaGL.mm

```diff
@@ -74,6 +75,9 @@
 }
 
 void PlatformCocoaGL::terminate() noexcept {
+    if (pImpl->mObserver) {
```

Comment:
I don't think we need to check if `mObserver` isn't `nil`. If it is, then `removeObserver:nil` will simply do nothing. Same with down below.

Reply:
I was wondering about that... thanks, fixed it up!